### PR TITLE
Clarify documentation for ReferenceEvaluator, fix type issue in Deform implementation

### PR DESCRIPTION
**Pull request by xadupre**
### Description
Extend the documentation for uses defining an implemenation for un updated operator.

### Motivation and Context
Documentation.

### CODE REVIEW

**Review by jcwchen — APPROVED** [review]
LGTM. Might be worthy to mention the Deform fix in the PR title.

File: onnx/reference/reference_evaluator.py [hunk 1]
```diff
@@ -143,6 +143,13 @@ class Conv(_Conv):
 
             def _run(self, ...):
                 ...
+
+        An operator may be different in a later opset. In that case,
+        a new implementation needs to be registered. `Pad_11`, `Pad_18`.
+        `Pad_11` is the implementation chose for opset in [11, 17].
+        `Pad_18` is selected for any greater opset. Both classes must be
+        important file `_op_list.py` to registered their existence to the
```
Comment (gramalingam) [inline onnx/reference/reference_evaluator.py:151]: "important" => "imported" ? 
"registered" => "register" ?

**Review by gramalingam — APPROVED** [review]
Minor typos mentioned above,

File: onnx/reference/reference_evaluator.py (hunk 1 above)
Comment (xadupre) [reply onnx/reference/reference_evaluator.py:151]: done

**Pull request merged**